truss-chains/truss_chains/public_types.py

```diff
@@ -28,7 +28,12 @@
 CpuCountT = Literal["cpu_count"]
 CPU_COUNT: CpuCountT = "cpu_count"
 
-_BASETEN_API_SECRET_NAME = "baseten_chain_api_key"
+
+# NOTE(Tyron): This is a secret that points to an API key value.
+CHAIN_API_KEY_SECRET_NAME = "baseten_chain_api_key"
+
```

Comment:
do we need two different variables here with the same value? can we reuse `CHAIN_API_KEY_NAME`?

Reply:
I wanted to but API keys don't allow underscores.

Reply:
ah oops didn't look close enough to realize its underscores vs dashes, all good